packages/@aws-cdk/aws-appmesh/test/integ.mesh.expected.json

```diff
@@ -650,7 +650,7 @@
               "VirtualService": {
                 "VirtualServiceName": {
                   "Fn::GetAtt": [
-                    "service27C65CF7D",
```

Comment:
Why did these change their IDs? Will this cause replacement for current customers of the library?

Reply:
They were previously created inline and I refactored them so I could reference them at multiple spots. This isn't needed anymore so I can take it out of the diff.